src/Extension/MemberExtension.php

```diff
@@ -39,4 +46,68 @@ public function getDefaultRegisteredMethod()
     {
         return $this->owner->RegisteredMFAMethods()->byId($this->owner->DefaultRegisteredMethodID);
     }
+
+    public function updateCMSFields(FieldList $fields)
+    {
+        $fields->removeByName(['DefaultRegisteredMethodID', 'HasSkippedMFARegistration', 'RegisteredMFAMethods']);
+
+        if (!$this->currentUserCanViewMFAConfig()) {
+            return $fields;
+        }
+
+        $fields->addFieldToTab(
+            'Root.Main',
+            $methodListField = RegisteredMFAMethodListField::create(
+                'MFASettings',
+                _t(__CLASS__ . '.MFA_SETTINGS_FIELD_LABEL', 'Multi-factor Authentication Settings (MFA)'),
```

Comment:
```suggestion
                _t(__CLASS__ . '.MFA_SETTINGS_FIELD_LABEL', 'Multi Factor Authentication settings (MFA)'),
```

Justification:

* Site config setting label is `Multi Factor Authentication (MFA)` already
* SS ^4.4 defaults to lowercase except the first word, so `settings` rather than `Settings` would fit nicer